styledComponents.js

```diff
@@ -1,4 +1,17 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
+
+export const StyledSignedOutContainer = styled.div`
+  opacity: 0.4;
+  pointer-events: none;
+
+  ${({ $isFlexProp }) =>
+    $isFlexProp &&
+    css`
+      display: flex;
+      flex-direction: column;
+      gap: 30px;
+    `}
+`;
```

Comment:
this apparently only changes the distance between the items of the children. Is this really necessary or can you just define one gap for both logged in and logged out state?

Reply:
Unfortunately, it is necessary because I have three elements on the home page, two of which I want to display grayed out when logged out. Without the flex properties, these elements are moved, because by adding the StyledSignedOutContainer as an additional element, the originally set flex properties only apply between the visible and the grayed out elements. At least, I don't know how else to do it. 😅

Reply:
I just noticed that I changed my code so that I don't use this component more than once. Then I can simply move it to the `index.js` of the `HomePage` and use it there without a prop query.